setup.sh

```diff
@@ -6,16 +6,39 @@ source ${BASH_SOURCE%/*}/vault_auth.sh
 echo "Installing Pip and required Python packages"
 apt-get install python3-pip -y > /dev/null
 pip3 install -q -r requirements.txt
+
+echo "-------------------------------------------"
+echo "Installing bb8 user and group"
+
 if ! id -u bb8 > /dev/null 2>&1; then
     useradd bb8
     password=$(vault read --field password secret/backup/bb8/user)
     echo "bb8:$password" | chpasswd
+
+    # set bb8 home dir
+    mkdir /var/lib/bb8/.ssh
+    usermod -m -d /var/lib/bb8 bb8
+    chown -R bb8:bb8 /var/lib/bb8
+
+    # add to docker group
+    usermod -aG docker bb8
```

Comment:
The script will fail if the docker group does not exist. Not a problem, but the requirement of docker and a docker group should be added to the README I think